### PR TITLE
LPS-66100 DB2 Fix

### PR DESCRIPTION
**Pull request by sergiogonzalez**

### CODE REVIEW

**Comment by sergiogonzalez** [comment]
ci:retest

File: portal-impl/src/META-INF/portal-model-hints.xml [hunk 1]
```diff
@@ -342,7 +342,7 @@
 		<field name="modifiedDate" type="Date" />
 		<field name="fileEntryTypeKey" type="String" />
 		<field localized="true" name="name" type="String">
-			<hint name="max-length">4000</hint>
+			<hint-collection name="CLOB" />
```
Comment (juangon) [inline portal-impl/src/META-INF/portal-model-hints.xml]: I remember some regressions caused by this kind of data type changes in Oracle, like https://issues.liferay.com/browse/LPS-47261. Are we using this field in any DISTINCT clause?

**Comment by brianchandotcom** [comment]
ci:retest

**Comment by brianchiu** [comment]
@brianchandotcom 
That workflowadmin poshi failure is OK. It was a fluke.

**Comment by brianchandotcom** [comment]
@sergiogonzalez when I run "ant build-upgrade-table", it modifies src/com/liferay/portal/upgrade/v7_0_0/util/DLFileEntryTypeTable.java

Your "upgrade.table.dir" is stale.

Mine points to

upgrade.table.dir=/Users/brian/dev/projects/github/upgrade

and that dir has:

-rwxrwxrwx    1 brian  staff  202666361 Mar  3  2010 liferay-portal-src-6.0.0.zip
-rwxrwxrwx    1 brian  staff  228371179 Apr 19  2010 liferay-portal-src-6.0.1.zip
-rwxrwxrwx    1 brian  staff  232711374 Nov 30  2010 liferay-portal-src-6.0.10.zip
-rwxrwxrwx    1 brian  staff  230320369 Jan 13  2011 liferay-portal-src-6.0.11.zip
-rwxrwxrwx    1 brian  staff  257214139 Jul 27  2011 liferay-portal-src-6.0.12.zip
-rwxrwxrwx    1 brian  staff  233014815 Jun  7  2010 liferay-portal-src-6.0.2.zip
-rwxrwxrwx    1 brian  staff  232678915 Jul 15  2010 liferay-portal-src-6.0.3.zip
-rwxrwxrwx    1 brian  staff  233996028 Jul 22  2010 liferay-portal-src-6.0.4.zip
-rwxrwxrwx    1 brian  staff  232230294 Oct 21  2010 liferay-portal-src-6.0.5.zip
-rwxrwxrwx    1 brian  staff  238836315 Feb 25  2011 liferay-portal-src-6.0.6.zip
-rwxrwxrwx    1 brian  staff  245322722 Jan  6  2012 liferay-portal-src-6.1.0.zip
-rwxrwxrwx    1 brian  staff  247858180 Feb 23  2012 liferay-portal-src-6.1.10.zip
-rw-r--r--@   1 brian  staff  297178145 Apr  7 13:15 liferay-portal-src-6.2.0.zip
-rw-r--r--@   1 brian  staff  297128649 Apr  7 13:15 liferay-portal-src-6.2.1.zip
-rw-r--r--@   1 brian  staff  312205741 Apr  7 13:14 liferay-portal-src-7.0.0.zip

You're missing liferay-portal-src-7.0.0.zip, can you resend? Thx.

**Pull request closed without merging**